gradle.properties

```diff
@@ -8,8 +8,8 @@ githubSlug=micronaut-projects/micronaut-graphql
 # Dependencies
 cglibVersion=3.3.0
 grailsVersion=3.3.10
-graphqlJavaVersion=14.0
-graphqlJavaToolsVersion=6.0.2
+graphqlJavaVersion=2020-07-03T00-47-06-d3032e7
+graphqlJavaToolsVersion=6.1.0
```

Comment:
@marceloverdijk same with this upgrades.

Reply:
@ilopmar same for GraphQL Java version. Latest official version is `15.0`. The `2020-07-03T00-47-06-d3032e7` seems some version automatically deployed after a commit or something?

PS: I was updating to GraphQL 15.0 a couple of weeks ago and I found that other libraries, like tools and SPQR were not compliant yet... meaning these demo's won't work anymore.

See also https://github.com/leangen/graphql-spqr/issues/356 and https://github.com/graphql-java-kickstart/graphql-java-tools/issues/405

At the end we should upgrade to 15 imo, even if it means the demo's are broken...

Reply:
@marceloverdijk Thanks for the clarification. What about `graphqlJavaToolsVersion=6.1.0`. Can we update it?

I'll close this PR and manually upgrade the `micronaut-gradle-plugins` and `graphqlJavaToolsVersion` if you confirm we can do it.

Reply:
Yes, upgrading `graphqlJavaToolsVersion=6.1.0` should be good 💪

Reply:
Thanks!